Intersect (Core)/Config/LootOptions.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Intersect.Config
+{
+    public class LootOptions
+    {
+
+        // Defines how long (in ms) an item drop will be ''owned'' by a player and their party.
```

Comment:
Proper XML doc for these please.

Also add a brief description to the class itself as a guideline of what sort of variables should or shouldn't be added to this scope in the future.

```
/// <summary>
/// Description here
/// </summary>
```

Reply:
Added some basic XML documentation to this class and its options.